src/react/boosts/BoostCard.tsx

```diff
@@ -19,94 +21,134 @@ export interface Items {
+              <p>{this.props.about}</p>
+            </BoostDescription>
+            <BoostDescriptionTable>
+              <h3>Where your support goes</h3>
+              <table>
+                <thead>
+                  <tr>
+                    <th>Item</th>
+                    <th>Cost</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {this.props.items.map((item: any, i: number) => (
+                    <tr key={i}>
+                      <td>{item.item}</td>
+                      <td>{"$" + item.cost}</td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
+            </BoostDescriptionTable>
+            <BoostDescription>
+              <h3>Questions about Boosts?</h3>
+              <p><a href="#TODO">Learn more in our FAQ</a></p>
+            </BoostDescription>
           </>
         )}
-      </BoostNewsroomInfo>
-      <BoostFlexStart>
-        <BoostProgressCol open={props.open}>
-          <BoostProgress open={props.open} goalAmount={props.goalAmount} paymentsTotal={props.paymentsTotal} daysLeft={boost.daysLeft} />
-        </BoostProgressCol>
-        {props.open && (<BoostButton onClick={props.handlePayments}>Support</BoostButton>)}
-      </BoostFlexStart>
-      {props.open && (
-        <>
-          <BoostDescription>
-            <p>{props.why}</p>
-          </BoostDescription>
-          <BoostDescription>
-            <h3>What the outcome will be</h3>
-            <p>{props.what}</p>
-          </BoostDescription>
-          <BoostDescription>
-            <h3>About the newsroom</h3>
-            <p>{props.about}</p>
-          </BoostDescription>
-          <BoostDescriptionTable>
-            <h3>Where your support goes</h3>
-            <table>
-              <thead>
-                <tr>
-                  <th>Item</th>
-                  <th>Cost</th>
-                </tr>
-              </thead>
-              <tbody>
-                {props.items.map((item: any, i: number) => (
-                  <tr key={i}>
-                    <td>{item.item}</td>
-                    <td>{"$" + item.cost}</td>
-                  </tr>
-                ))}
-              </tbody>
-            </table>
-          </BoostDescriptionTable>
-          <BoostDescription>
-            <h3>Questions about Boosts?</h3>
-            {/* TODO(sruddy) add FAQ URL */}
-            <p><a href="">Learn more in our FAQ</a></p>
-          </BoostDescription>
-        </>
-      )}
-    </BoostWrapper>
-  );
+      </BoostWrapper>
+    );
+  };
+
+  private daysLeft = (dateEnd: string) => {
+    const endDate = Date.parse(dateEnd);
+    const currentDate = Date.now();
+    const daysLeftNum = Math.trunc((endDate - currentDate) / (24 * 60 * 60 * 1000));
+    let daysLeft;
+
+    if (daysLeftNum === 1) {
+      daysLeft = "1 day left";
+    } else if (daysLeftNum < 1) {
+      daysLeft = "Ends today";
```

Comment:
Not necessary for MVP but I think we want an hour or minute countdown for less than a day? I'd put a note somewhere, either a todo comment or .5 point jira ticket.

Reply:
I meant to ask design about this I added a TODO for now